tests/standalone/six.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
```

Comment:
You need that absolute_import only due to your strange naming of the test as "six", why not go with SixUsing.py to match what other tests have.

Reply:
Yeah, Sure!

Reply:
Won't you delete this file then, it seems you forgot about that?